controller/controller.go

```diff
@@ -406,9 +389,10 @@ func (c *Controller) reconcile(ctx context.Context, req reconcile.Request) (reco
 		ctx = setLoggerCtxValue(ctx, loggerKeyObject, m.GetSelfLink())
 		ctx = setLoggerCtxValue(ctx, loggerKeyVersion, m.GetResourceVersion())
 
-		c.logger.LogCtx(ctx, "level", "debug", "message", "reconciling object")
-		c.deleteFunc(ctx, obj)
-		c.logger.LogCtx(ctx, "level", "debug", "message", "reconciled object")
```

Comment:
I dropped a couple of logs that I thought became irrelevant over time. This was more noise than signal.